### DISCUSSION TITLE
AI能一分鐘產出四篇新聞？！

### DISCUSSION THREAD

**Discussion by 20010906** [1101-GS4719-C/Python-Community-GS4719-C#311] [General]
由實驗可知，AI撰寫之新聞不僅內容合理，符合邏輯可信度甚至高於人類，相信在未來AI能有助於提高寫作效率，但同時也會產生濫用等等的行為。
[AI撰寫新聞](https://tw-news-yahoo-com.cdn.ampproject.org/v/s/tw.news.yahoo.com/amphtml/ai%E6%96%B0%E8%81%9E%E5%B9%BE%E5%8F%AF%E4%BA%82%E7%9C%9F-1%E5%88%86%E9%90%984%E7%AF%87%E7%94%A2%E8%83%BD%E8%B6%85%E9%AB%98-022632975.html?amp_js_v=a6&amp_gsa=1&usqp=mq331AQKKAFQArABIIACAw%3D%3D#aoh=16416576537305&referrer=https%3A%2F%2Fwww.google.com&amp_tf=%E4%BE%86%E6%BA%90%EF%BC%9A%251%24s&ampshare=https%3A%2F%2Ftw.news.yahoo.com%2Fai%25E6%2596%25B0%25E8%2581%259E%25E5%25B9%25BE%25E5%258F%25AF%25E4%25BA%2582%25E7%259C%259F-1%25E5%2588%2586%25E9%2590%25984%25E7%25AF%2587%25E7%2594%25A2%25E8%2583%25BD%25E8%25B6%2585%25E9%25AB%2598-022632975.html)

**Comment by 109802503gloria** [comment]
有這樣的技術，網路上的資訊要更加小心的篩選了

**Comment by chun1203** [comment]
沒想到AI寫的新聞甚至可以騙過專業人士 太厲害啦

**Comment by 107602505** [comment]
AI生成文章雖然可以輔助人們提高寫作的效率，但我認為也不應該太依賴它不然可能會漸漸失去獨立思考的能力

**Comment by MAKERRRofKEVIN** [comment]
對時事寫手來說會是危機同時也是福音，需要擔心的是相當一部分人的工作會被取代，喜的是過去繁瑣的工作能夠被簡化成校稿、潤筆等工作，對讀者來說最需要擔心的可能是這類的工具是否會被用來散佈假新聞，特別是機器本身是沒有能力去判斷自己真的寫出了什麼

**Comment by ChinChun0220** [comment]
文末有提到，這樣未來學生的作業的寫手可能都不用是真的一個人了…這種作弊方法防不勝防

**Comment by mesakiyuta** [comment]
AI生成的文章跟人寫的文章相差不多，且生產速度極快，只要訓練資料有偏頗，是不是就會大量產生具有立場的文章? 感覺今後需要有更好的辨識能力。

**Comment by cody910120** [comment]
看來以後新聞要嘛賊藍要嘛賊綠了哭阿

**Comment by Joan0518** [comment]
加上前幾篇AI記者的文章，以後真的是人工智慧的時代

**Comment by 109302013evantsao** [comment]
我想說個好笑的，事實上記者有可能生產新聞比這個還快，因為我常常看到Dcard上的文章直接轉貼到新聞上，複製貼上，就是這麼簡單

**Comment by www0000596** [comment]
雖然AI可以用比較高的效率產出新聞，但每家新聞台、電視台出的新聞多少還是會有自家的色彩
我認為內容還是需要靠大家的媒體識讀能力來分辨內容的合理性吧🧐

**Comment by chupochun** [comment]
在發展AI的同時也要規範使用範圍才不會造成濫用

**Comment by pekoranoinu** [comment]
AI真的是不能隨便濫用啊!
